app/models/katello/host/subscription_facet.rb

```diff
@@ -34,6 +35,19 @@ def import_database_attributes(consumer_params)
         end
       end
 
+      def update_consumer_type(consumer_params)
+        if consumer_params.try(:[], 'type').try(:[], 'label') == 'hypervisor'
+          self.consumer_type = 'hypervisor'
+        elsif consumer_params.try(:[], 'facts').try(:[], 'virt.is_guest') == "true"
```

Comment:
Where does the uuid field come from at this point? If I am client registering via sub-man, what is sending the VM uuid? (not to be confused with the subscription facet uuid coming from Candlepin)

Also, if we have the uuid to indicate its a VM, thats one thing. But are there existing ways for a user to search? e.g. `virtual = true` or `physical = true` ?